ui/app/AppLayouts/Onboarding2/UseRecoveryPhraseFlow.qml

```diff
@@ -15,7 +12,7 @@ SQUtils.QObject {
         Login
     }
 
-    required property StackView stackView
+    /*required*/ property int type
```

Comment:
Why not `required`?

Reply:
Because of https://bugreports.qt.io/browse/QTBUG-84269. I checked and it works ok on Qt 6. Will add comment there.